.env

```diff
@@ -0,0 +1,7 @@
+APP_ENV=dev
```

Comment:
please add `.env` to the `.gitignore` file (keeping only `.env.dist` will be enough)